Example/Tests/GINIMultipageReviewControllerTests.swift

```diff
@@ -0,0 +1,187 @@
+        let thirdCell = vc.collectionView(vc.bottomCollection,
+                                          cellForItemAt: IndexPath(row: 2, section: 0)) as? MultipageReviewBottomCollectionCell
+        XCTAssertEqual(firstCell?.documentImage.image, imageDocuments[0].previewImage,
+                       "First cell image should match the one passed in the initializer")
+        XCTAssertEqual(firstCell?.pageIndicator.text, "1",
+                       "First cell indicator should match its position")
+        XCTAssertEqual(secondCell?.documentImage.image, imageDocuments[1].previewImage,
+                       "Second cell image should match the one passed in the initializer")
+        XCTAssertEqual(secondCell?.pageIndicator.text, "2",
+                       "Second cell indicator should match its position")
+        XCTAssertEqual(thirdCell?.documentImage.image, imageDocuments[2].previewImage,
+                       "Third cell image should match the one passed in the initializer")
+        XCTAssertEqual(thirdCell?.pageIndicator.text, "3",
+                       "Third cell indicator should match its position")
+        XCTAssertEqual(firstCell?.documentImage.contentMode, UIViewContentMode.scaleAspectFill,
+                       "First cell content mode should match the one passed in the initializer")
+    }
+    
+    func testBottomCollectionCellSize() {
+        let vc = MultipageReviewController(imageDocuments: imageDocuments)
+        _ = vc.view
+        vc.view.setNeedsLayout()
+        vc.view.layoutIfNeeded()
+        
+        let firstCellIndexPath = IndexPath(row: 0, section: 0)
+        let cellSize = vc.collectionView(vc.bottomCollection,
+                                         layout: vc.bottomCollection.collectionViewLayout,
+                                         sizeForItemAt: firstCellIndexPath)
+        
+        let size = MultipageReviewBottomCollectionCell.size
+        
+        XCTAssertEqual(cellSize, size,
+                       "Bottom collection cells should have the value declared in the class")
+    }
+    
+    func testBottomCollectionInsets() {
+        let collectionInsets = vc.collectionView(vc.bottomCollection,
+                                                 layout: vc.bottomCollection.collectionViewLayout,
+                                                 insetForSectionAt: 0)
+        
+        XCTAssertEqual(collectionInsets, vc.bottomCollectionInsets,
+                       "Main collection insets should be zero")
+    }
+    
+    func testToolBarItemsOnInitialization() {
+        XCTAssertEqual(self.vc.toolBar.items![0], self.vc.rotateButton,
+                       "First toolbar item should be the rotateButton")
+        XCTAssertEqual(self.vc.toolBar.items![2], self.vc.reorderButton,
+                       "Third toolbar item should be the rotateButton")
+        XCTAssertEqual(self.vc.toolBar.items![4], self.vc.deleteButton,
```

Comment:
I think that if the item is null, an exception will be thrown and it will be a bit more difficult to follow than a plain old assertion failure (the assertion failure will print the "Fifth toolbar item should be the rotateButton" description). So, I'd personally choose to avoid risking Optional-related exceptions.

Reply:
Good point, I thought that it will show an optional related exceptions, instead of assert message.